src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -221,16 +267,18 @@ export class ListingCreateComponent implements OnInit {
         });
     }
 
-    afterClosed(confirmed) {
-        if (confirmed) {
-            this.createNotes();
-        }
-        if (this.editMode) {
-            this.afterEdit();
+    private prepareListingTypeData() {
+        switch (this.chosenListingType) {
+            case null:
```

Comment:
*case null* ? is it valid? should we use case *default*

Reply:
null is valid, but 'default' matches more cases so is better.